create_offline_installer.py

```diff
@@ -564,7 +586,7 @@ def build(self):
         print('##[endgroup]')
 
         print('##[group]Pin python version in the installed conda environment', flush=True)
-        self.pin_python_version('3.7')
+        self.pin_python_version()
```

Comment:
how come the version isn't pinned anymore?

Reply:
it is, see line 472 above :)